### PR TITLE
Created an empty `@solana/react` package

### PR DESCRIPTION
**Pull request by steveluscher**

### CODE REVIEW

**Comment by steveluscher** [comment]
* **#2772** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2772?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2791** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2791?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2790** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2790?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @steveluscher and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .changeset/pre.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@solana/web3.js": "0.0.0-development",
     "@solana/options": "2.0.0-preview.2",
     "@solana/programs": "2.0.0-preview.2",
+    "@solana/react": "2.0.0-preview.3",
```
Comment (buffalojoec) [inline .changeset/pre.json:23]: Should this be `3`?

**Review by buffalojoec — APPROVED** [review]
Lgtm. Looks like for CI you can temporarily add `--passWithNoTests` or maybe a `todo`?

File: .changeset/pre.json (hunk 1 above)
Comment (steveluscher) [reply .changeset/pre.json:23]: They should all be 3. I forgot to advance these again after the last publish.

**Comment by steveluscher** [comment]
### Merge activity

* **Jun 11, 12:22 AM PDT**: @steveluscher started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2790).

**Pull request merged**